### ISSUE TITLE
Consider renaming the project to 'bazel-maven-deps'

### ISSUE THREAD

**Issue by cgrushko** [johnynek/bazel-deps#64]
I think 'bazel-manve-deps' (or something with "maven" in it) will help users discover this tool.
Sorry if I'm a bit intrusive :)

CC @googlestanke

**Comment by johnynek** [comment]
Yeah. I agree. I actually forked from a previous project and then rewrote the whole thing. Another reason to rename.

**Comment by davidstanke** [comment]
:) Yep, +1 to renaming to include Maven.

**Comment by johnynek** [comment]
here is where I asked how to do this:

https://twitter.com/posco/status/912475560911048704

I know how to rename the project. I don't know how to break the link to the original.
